commit 70ff2db7ea29928dbf2ef9cbd9b82901908d21ee
Author: Todd Whiteman <todd.whiteman@joyent.com>
Date:   2019-05-07T09:09:28-07:00 (5 months ago)
    
    TRITON-1647 need to save vmapi runtests tap output
    Reviewed by: Trent Mick <trentm@gmail.com>
    Approved by: Trent Mick <trentm@gmail.com>

diff --git a/test/runtests b/test/runtests
index 9960a0c..555a2f1 100755
--- a/test/runtests
+++ b/test/runtests
@@ -210,5 +210,5 @@ fi
 if [[ -n "$test_files" ]]; then
     VMAPI_IP=$VMAPI_IP CNAPI_IP=$CNAPI_IP NAPI_IP=$NAPI_IP \
     PATH=$NODE_INSTALL/bin:$PATH $NODEUNIT --reporter \
-    $opt_reporter $test_files
+    $opt_reporter $test_files | tee $OUTPUT_DIR/vmapi.tap
 fi
